README.md

```diff
@@ -1,4 +1,4 @@
-# Cloud Foundry CDN Service Broker [![Build Status](https://travis-ci.org/18F/cf-cdn-service-broker.svg?branch=master)](https://travis-ci.org/18F/cf-cdn-service-broker)
+# Cloud Foundry CDN Service Broker [![Build Status](https://travis-ci.org/cloud-gov/cf-cdn-service-broker.svg?branch=master)](https://travis-ci.org/cloud-gov/cf-cdn-service-broker)
```

Comment:
Can we just drop the build status tag since we don't use Travis?

Reply:
This is just a tag in the README that shows whether your Travis CI build on `master` is passing. But we're definitely not using Travis. And the link goes to this broken link: https://app.travis-ci.com/18F/cf-cdn-service-broker